fsd_utils/config/commonconfig.py

```diff
@@ -282,6 +279,36 @@ class CommonConfig:
             "section_weighting": None,
         },
     )
+
+    # ---------------
+    #  Fund Config
+    # ---------------
+
+    COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
+    COF_ROUND_2_ID = "c603d114-5364-4474-a0c4-c41cbf4d3bbd"
+    COF_ROUND_2_W3_ID = "5cf439bf-ef6f-431e-92c5-a1d90a4dd32f"
+    DEFAULT_FUND_ID = COF_FUND_ID
+
+    @classmethod
+    def get_default_round_id(cls):
+        try:
+            r2_w3 = get_remote_data_as_json(
+                cls.FUND_STORE_API_HOST
+                + cls.ROUND_ENDPOINT.format(
+                    fund_id=cls.COF_FUND_ID, round_id=cls.COF_ROUND_2_W3_ID
+                )
+            )
+            cof_r2_w3_is_open = current_datetime_after_given_iso_string(
+                r2_w3["opens"]
+            )
+
+            if cof_r2_w3_is_open:
+                return cls.COF_ROUND_2_W3_ID
+            else:
+                return cls.COF_ROUND_2_ID
+        except Exception as e:  # noqa:F841
+            return cls.COF_ROUND_2_ID
```

Comment:
Is there any chance of making this a more specific `Exception` or is it just a catchall?

Reply:
After seeing how this method is used on the other PRs, I think it would be good to log the errors that happen here especially if it's generic, since it seems it's called once (and only once) - for example if some network error occurs then the micro-service that encountered said exception might be stuck with `COF_ROUND_2_ID` until a reboot, given this will only be relevant if `cof_r2_w3_is_open` & we're expecting `COF_ROUND_2_W3_ID`